content/21.nutraceuticals.md

```diff
@@ -274,65 +280,73 @@ Other studies are investigating whether probiotics may affect patient outcomes f
 
 Generally, the efficacy of probiotic use is a controversial topic among scientists.
 In Europe, EFSA has banned the term probiotics on products labels, which has elicited either criticism for EFSA or support for probiotics from researchers in the field [@doi:10.1016/j.bpg.2015.12.001; @doi:10.1017/S000711451100287X; @doi:10.1038/nrgastro.2014.66].
-This is due to the hyperbolic claims placed on the labels of various probiotic products, which lack rigorous scientific data to support their efficacy.
-Overall, the data supporting probiotics in the treatment or prevention of many different disorders and diseases is not conclusive as the quality of the evidence is generally considered low [@doi:10.1007/s10096-014-2086-y].
+This regulation is due to the hyperbolic claims placed on the labels of various probiotic products, which lack rigorous scientific data to support their efficacy.
+Overall, the data supporting probiotics in the treatment or prevention of many different disorders and diseases is not conclusive, as the quality of the evidence is generally considered low [@doi:10.1007/s10096-014-2086-y].
 However, in the case of probiotics and respiratory infections, the evidence seems to be supportive of their potential therapeutic value.
 Consequently, several investigations are underway to investigate the prophylactic and therapeutic potential of probiotics for COVID-19.
-However, blind use of conventional probiotics for COVID-19 is cautioned against until the pathogenesis of SARS-CoV-2 is further established [@doi:10/d2qq].
+The blind use of conventional probiotics for COVID-19 is currently cautioned against until the pathogenesis of SARS-CoV-2 can be further established [@doi:10/d2qq].
 Until clinical trials investigating the prophylactic and therapeutic potential of probiotics for COVID-19 are complete, it is not possible to provide an evidence-based recommendation for their use.
 Despite these concerns, complementary use of probiotics as an adjuvant therapeutic has been proposed by the Chinese National Health Commission and National Administration of Traditional Chinese Medicine [@doi:10.1111/1751-2980.12851]. 
+While supply issues prevented the probiotics market from showing the same rapid response to the COVID-19 as some other supplements, many suppliers are reporting growth during the pandemic [@url:https://www.bloomberg.com/press-releases/2020-08-03/probiotics-market-worth-76-7-billion-by-2027-exclusive-report-covering-pre-and-post-covid-19-market-analysis-by-meticulous].
+Therefore, the public response once again seems to have adopted supplements promoted as bolstering the immune response despite a lack of evidence suggesting they are beneficial for preventing or mitigating COVID-19.
 
 ### Discussion
 
+In this review, we report the findings to date of analyses of several dietary supplements and nutraceuticals.
```

Comment:
My PhD advisor always liked me to start the discussion with the paragraph that exactly addressed what the paper showed and then move on to look at it in context -- I don't know what you think about moving this paragraph up!

Reply:
In hindsight I agree with you, a wise supervisor!